### PR TITLE
Create gremlin docker

### PR DESCRIPTION
**Pull request by jllopes**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+              docker login -u $DOCKER_USER -p $DOCKER_PASS
+              docker tag codacy/${CIRCLE_PROJECT_REPONAME}:latest codacy/$CIRCLE_PROJECT_REPONAME:1.0.$CIRCLE_BUILD_NUM
+              docker push codacy/$CIRCLE_PROJECT_REPONAME:1.0.$CIRCLE_BUILD_NUM
+              docker push codacy/$CIRCLE_PROJECT_REPONAME:latest
+            fi
+workflows:
+  version: 2
+  build-and-deploy:
+    jobs:
+      - build
```
Comment (machadoit) [inline .circleci/config.yml]: Add newline at the end of the file

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    && unzip apache-tinkerpop-gremlin-server-$GREMLIN_VERSION-bin.zip \
+    && mv apache-tinkerpop-gremlin-server-$GREMLIN_VERSION tinkergraph \
+    && rm -rf apache-tinkerpop-gremlin-server-$GREMLIN_VERSION-bin.zip \
+    && cd tinkergraph \
+    && mkdir db \
+    && /bin/bash ./bin/gremlin-server.sh install org.apache.tinkerpop neo4j-gremlin $GREMLIN_VERSION \
+    && rm -rf javadocs \
+    && rm -rf docs \
+    && cd .. \
+    && /bin/bash ./validate.sh
```
Comment (machadoit) [inline Dockerfile]: you don't need ./, you can just /bin/bash validate.sh

File: grapeConfig.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+	  <filesystem name="cachedGrapes">
+		<ivy pattern="${user.home}/.groovy/grapes/[organisation]/[module]/ivy-[revision].xml"/>
+		<artifact pattern="${user.home}/.groovy/grapes/[organisation]/[module]/[type]s/[artifact]-[revision](-[classifier]).[ext]"/>
+	  </filesystem>
+	  <ibiblio name="localm2" root="file:${user.home}/.m2test/repository/" checkmodified="true" changingPattern=".*" changingMatcher="regexp" m2compatible="true" usepoms="false"/>
+	  <ibiblio name="jcenter" root="https://jcenter.bintray.com/" m2compatible="true"/>
+	  <ibiblio name="ibiblio" m2compatible="true"/>
+	</chain>
+  </resolvers>
+</ivysettings>
```
Comment (machadoit) [inline grapeConfig.xml]: newline at the end

File: validate.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```
Comment (machadoit) [inline validate.sh]: A echo at the begging wouldn't hurt, like echo "====> Checking if docker build successfully" or something like that

**Review by machadoit — CHANGES_REQUESTED** [review]

**Comment by rtfpessoa** [comment]
@jllopes seems like you did not

**Review by rtfpessoa — DISMISSED** [review]

**Review by machadoit — DISMISSED** [review]

**Review by machadoit — APPROVED** [review]

**Review by rtfpessoa — APPROVED** [review]

**Pull request merged**